yarn.lock

```diff
@@ -13248,13 +13254,6 @@ minipass@^3.0.0, minipass@^3.1.1:
   dependencies:
     yallist "^4.0.0"
 
-minipass@^4.0.0:
```

Comment:
This was probably left off of another PR.

Reply:
need to rebase? or do you mean like we merged this and it shouldn't be there?

Reply:
It is a leftover from #6492